src/App.css

```diff
@@ -166,3 +177,17 @@
   color: red;
   font-size: 0.7rem;
 }
+.history-list-content>label{
+  font-size: 0.85rem;
+}
+/*@Shreeti can be removed ** START** just added for the heading for Vaccination*/
+.history-list-content:first-of-type{
+
+  background-color: transparent;
+}
+.history-list-content:first-of-type::after {
+  content: " Vaccination History";
```

Comment:
strange space in content, it's better to add padding-left 1-2 pixels if needed and remove the space

Reply:
since we don't have a page for vaccination yet, only the space has been removed for now. necessary change will be made upon the page's implementation